src/main/java/com/example/demo/services/ProductService.java

```diff
@@ -0,0 +1,11 @@
+package com.example.demo.services;
+
+import com.example.demo.database.Product;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping("/api/product")
+public class ProductService  {
```

Comment:
You should have all business logic in the service layer, as discussed at the meeting